packit_service/worker/build/copr_build.py

```diff
@@ -371,7 +371,6 @@ def run_build(
             build = self.api.copr_helper.copr_client.build_proxy.create_from_file(
                 ownername=owner, projectname=self.job_project, path=self.srpm_path
             )
-            self.srpm_model.set_url(build.source_package.get("url"))
```

Comment:
We can update it multiple times. => Do we need to remove this?

Reply:
it was 404 even during the build

Reply:
OK, I thought it can't harm anything but since we don't show anything for `None`, it's probably better to not set the URL here.